pinot-core/src/main/java/com/linkedin/pinot/core/query/pruner/PartitionSegmentPruner.java

```diff
@@ -32,6 +34,7 @@
  * Implementation of {@link SegmentPruner} that uses partition information to perform pruning.
  */
 public class PartitionSegmentPruner extends AbstractSegmentPruner {
+  public static final String IN_CLAUSE_DELIMITER = "\t\t";
```

Comment:
We are able to do a query with 35 items and 60 features in about 80 ms vs more than 5 seconds earlier